components/definitions/newTab.d.ts

```diff
@@ -39,7 +39,10 @@ declare namespace NewTab {
     destinationUrl: string
   }
 
+  export type BrandedWallpaperType = 'image' | 'richMedia'
```

Comment:
if possible it'd be great to not define this here

Reply:
Removed!